stable/fluent-bit/templates/config.yaml

```diff
@@ -89,15 +89,20 @@ data:
         Match *
         Host  {{ .Values.backend.es.host }}
         Port  {{ .Values.backend.es.port }}
-        Logstash_Format On
         Retry_Limit False
         Type  {{ .Values.backend.es.type }}
 {{- if .Values.backend.es.time_key }}
         Time_Key {{ .Values.backend.es.time_key }}
 {{- end }}
+{{- if .Values.backend.es.include_tag_key }}
+        Include_Tag_Key {{ .Values.backend.es.include_tag_key }}
+{{- end }}
 {{- if .Values.backend.es.replace_dots }}
         Replace_Dots {{ .Values.backend.es.replace_dots }}
 {{- end }}
+{{- if .Values.backend.es.logstash_format }}
+        Logstash_Format {{ .Values.backend.es.logstash_format }}
+{{- end }}
```

Comment:
I'm kind of reluctant to add new options. I believe all of these can be set currently with 
.Values.extraEntries.output

Is that sufficient?

Reply:
@kfox1111 that's fine, but it should be consistent then. Logstash_Format was being provided a hardcoded default value (see your previous remark above, where I transformed it into this). I'm not introducing new options, I'm just making sure that the config doesn't statically provide values for me.